### DISCUSSION TITLE
addon-controls: Is there any way to add controls without them being added to the prop table?

### DISCUSSION THREAD

**Discussion by ByeBye-Sama** [storybookjs/storybook#11236] [Help]
I'm testing storybook v6 and while there is a way to make a prop from the prop table not have a control, I did not find anywhere in the documentation how to add a control without it being automatically added in the prop table

**Comment by shilman** [comment]
If you're using MDX, you can do `<Props of={MyComponent} />` which won't show the controls (`<Props story="storyName" />` does).

If you're using CSF + DocsPage, the only way is to not use args-stories. We could probably add a flag for that.

**Reply by ByeBye-Sama** [reply]
Thanks for the answer @shilman , ya I think a flag would be great, I notice other problem too when I try to edit props table with argstype if I put that all props have null controls the prop table doesn't change, but if I add any control for one of the props all the table is modified correctly. I would like to modify props table content with argstype but with all controls on null.

**Reply by shilman** [reply]
> I would like to modify props table content with argstype but with all controls on null.

I didn't understand this. can you give an example to help explain?

**Reply by ByeBye-Sama** [reply]
Sorry at some point it was solved and now I can't replicate it. Thanks for the help🙇‍♂️ . Btw why the style of controls look like this?

<img width="574" alt="Screen Shot 2020-06-19 at 15 03 06" src="https://user-images.githubusercontent.com/19353687/85175942-45fac400-b23e-11ea-907b-875fba4ddc13.png">

Meanwhile, knobs style look like

<img width="1209" alt="Screen Shot 2020-06-19 at 15 04 07" src="https://user-images.githubusercontent.com/19353687/85176137-b6a1e080-b23e-11ea-8d16-a1bf3f48790f.png">

Is any additional configuration I need to do? 🤔 Thanks for your time @shilman .

**Reply by shilman** [reply]
Fixed in https://github.com/storybookjs/storybook/pull/11216

**Comment by ndelangen** [comment]
It's likely not using the form components in lib/components (@storybook/components)

Would you like to help us set this right @ByeBye-Sama?

**Reply by ByeBye-Sama** [reply]
I wish, but I don't have the time, sorry :(
